Cargo.toml

```diff
@@ -16,3 +16,5 @@ itertools = "~0.9"
 semver = "~0.10"
 structopt = "~0.3"
 thiserror = "~1"
+difference = "~2"
+proptest = "~0.10"
```

Comment:
Proptest? :-O

I mean, it's a fine crate, sure, but I don't see it being used, I don't see any tests and I think it should go to de-dependencies at that point.